src/easing/easing.coffee

```diff
@@ -1,8 +1,8 @@
-bezier      = require './bezier-easing'
-PathEasing  = require './path-easing'
-mix         = require './mix'
-h           = require '../h'
-approximate = require('./approximate').default or require('./approximate');
+h           = require 'src/h'
+bezier      = require 'easing/bezier-easing'
+PathEasing  = require 'easing/path-easing'
+mix         = require 'easing/mix'
+approximate = require('easing/approximate').default or require('easing/approximate');
```

Comment:
@xavierfoucrier Do you know why we have a conditional require here? Seems unnecessary

Reply:
Need to have a look at it, but yeah it looks unnecessary for me :thinking: